middlewares/fileUploader.middleware.js

```diff
@@ -0,0 +1,20 @@
+const multer = require("multer");
+
+const avatarUploader = () => {
+  const storage = multer.diskStorage({
+    destination: function (req, file, cb) {
+      cb(null, "tmp");
+    },
+    filename: function (req, file, cb) {
+      const fileType = file.mimetype.split("/")[1];
+      if (fileType !== "jpg" && fileType !== "jpeg" && fileType !== "png") return cb(new Error("File must be a photo"));
```

Comment:
move all file formats to config as an array 
and check in array includes fileType